src/app/_services/container/container.service.spec.ts

```diff
@@ -0,0 +1,17 @@
+import {TestBed, inject} from '@angular/core/testing';
+
+import {ContainerService} from './container.service';
+import {HttpClient, HttpClientModule} from '@angular/common/http';
+
+describe('ContainerService', () => {
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [HttpClientModule],
+            providers: [ContainerService]
+        });
+    });
+
+    it('should be created', inject([ContainerService], (service: ContainerService) => {
```

Comment:
The tests requirement has been met :+1: , not sure if it is even possible to test it further than this.